src/layouts/Contact-HEDA Contact Layout.layout

```diff
@@ -322,6 +322,13 @@
         <fields>Grade__c</fields>
         <relatedList>Course_Enrollment__c.Contact__c</relatedList>
     </relatedLists>
+    <relatedLists>
```

Comment:
I don't think we want to add the related list by default but rather call out in the docs, "if you plan to leverage the new language tracking functionality, you will want to add the Contact Language related list to the HEDA Contact Page Layout"

Reply:
@juliesnow ^^please see above comment.

Reply:
@lifewithryan I just reviewed this with @aligeez  and @JasonFitzy yesterday so I'm tagging them here - should the Contact Language related list be added to the HEDA Contact page layout by default, or should it be a documentation item for them to add it if they want it?

Reply:
It sounds like we should add them by default. That is the precedence that has been set. Page Layouts apparently don't get push upgraded so the additions will only apply to new installations. Existing customers will have to add them on their own as well as setup FLS for new fields on existing objects, etc. Things that we need to call out in Docs. cc: @juliesnow

Reply:
Ignore my first comment up there about not including it...I was w-ww-wro--- I was wwwroo....I was wrong...